cassandra/components.py

```diff
@@ -789,6 +793,82 @@ def run_component(self):
         return 0
 
 
+class HectorStubComponent(ComponentBase):
+    """Component to serve Hector output data for RCP scenarios.
+
+    In cases where Hector output will be used only for the standard RCP
+    scenarios, it is not necessary to run the model, as the outputs never
+    change.  This module provides a way to read some of the most commonly used
+    outputs for those scenarios.
+
+    The component provides three capabilities:
+      * Tgav   : global mean temperature
+      * atm-co2: atmospheric CO2 concentration
+      * Ftot   : total radiative forcing
+
+    Each capability returns a data fram with data from all of the scenarios
+    specified in the configuration.  Spinup time steps are not included.
+
+    The parameters accepted by this component are:
+
+    scenarios : comma separated list of scenarios to include
+                e.g.: rcp26,rcp45
+                If omitted, all four rcp scenarios are included.
+
+    """
+
+    def __init__(self, cap_tbl):
+        super(HectorStubComponent, self).__init__(cap_tbl)
+        self.addcapability('Tgav')
+        self.addcapability('atm-co2')
+        self.addcapability('Ftot')
+
+
+    def run_component(self):
+        """Run the HectorStub component
+        
+        Load the requested scenarios and make each variable available to the
+        rest of the system.
+
+        """
+
+        import pandas as pd
+
+        scenarios = [x.strip() for x in self.params['scenarios'].split(',')]
```

Comment:
Note that the `ConfigReader` will automatically split comma-delimited parameters. So if the file has:
```
[HectorStubComponent]
scenarios = rcp26,rcp60
```
the value of the component's `params` dict will be `{'scenarios' : ['rcp26', 'rcp60']}`.

I'd suggest removing this line over the other solution which is to insist that the config parameters be quoted, eg:
```
[HectorStubComponent]
scenarios = "rcp26,rcp60"
```

Reply:
```suggestion
```
Good to know, thanks.